File: ts/src/leetcode/3110_score_of_string.ts
function scoreOfString(s: string): number {
  let score = 0;

  for (let i = 1; i < s.length; i++) {
    const sum = s.charCodeAt(i) - s.charCodeAt(i - 1);
    score += Math.abs(sum);
  }

  return score;
}

export default scoreOfString;

File: swift/3110_score_of_string.swift
class Solution {
    func scoreOfString(_ s: String) -> Int {
        var score = 0
        for (charA, charB) in zip(s, s.dropFirst()) {
            let difference = Int(charA.asciiValue!) - Int(charB.asciiValue!)
            score += abs(difference)
        }

        return score
    }
}
